cypress/locators/simple-color-picker/index.js

```diff
@@ -1,8 +1,11 @@
-import { SIMPLE_COLOR_PICKER_PREVIEW } from './locators';
+import { SIMPLE_COLOR_PICKER_PREVIEW, EXPERIMENTAL_SIMPLE_COLOR_PICKER } from './locators';
 
 // component preview locators
 export const simpleColorPickerPreview = () => cy.iFrame(SIMPLE_COLOR_PICKER_PREVIEW);
 export const simpleColorPickerInput = index => cy.iFrame(SIMPLE_COLOR_PICKER_PREVIEW)
   .find(`li:nth-child(${index}) > div > input`);
 export const simpleColorPickerDiv = index => cy.iFrame(SIMPLE_COLOR_PICKER_PREVIEW)
   .find(`li:nth-child(${index}) > div > div > span`);
+export const experimentalSimpleColorPickerInput = index => cy.iFrame(
+  EXPERIMENTAL_SIMPLE_COLOR_PICKER,
+).find(`div[data-component="simple-color"]:nth-child(${index}) > input`);
```

Comment:
this locator needs to be moved to locators.js file

Reply:
fixed